metrics.go

```diff
@@ -538,9 +538,8 @@ type KeyRotationInfo struct {
 }
 
 type CatalogInfo struct {
-	LastBucketScanned string `json:"lastBucketScanned"`
+	Bucket            string `json:"bucket"`
 	LastObjectScanned string `json:"lastObjectScanned"`
-	LastBucketMatched string `json:"lastBucketMatched"`
```

Comment:
I think better to deprecate existing and add new fields to avoid a major version bump

Reply:
Resolved

Reply:
@allanrogerr You need to regenerate. Your change isn't in the serializer code.

Reply:
@klauspost Resolved. Thank you!

Reply:
Seems like the CI check was broken: https://github.com/minio/madmin-go/pull/407